AUTHORS.txt

```diff
@@ -215,242 +324,134 @@ Sean Sun <pinggodstudio@gmail.com>
-sojournerc <cmeyer@zvelo.com>
-sorrycc <sorrycc@gmail.com>
-swindme <swindme@163.com>
-syssam <s.y.s.sam.sys@gmail.com>
-thegatheringstorm <tgs@tgs.blue>
-tianli.zhao <275287902@qq.com>
-tom <caolvchong@gmail.com>
-twobin <twobin@live.com>
-u3u <qwq@qwq.cat>
-ustccjw <317713370@qq.com>
-valleykid <valleykiddy@gmail.com>
-vgeyi <vgeyiz@126.com>
-wangtao0101 <yuecjn@gmail.com>
-wangxiaolei <fatelei@gmail.com>
-warmhug <hualei5280@gmail.com>
-whtang906 <whtang906@gmail.com>
-wizawu <wizawu@gmail.com>
-wonyun <wy393767068@163.com>
-xiaofan2406 <xiaofan2406@gmail.com>
-yangwukang <yangwukang@boco.com.cn>
-yeliex <yeliex@yeliex.com>
-yiminghe <yiminghe@gmail.com>
-yociduo <yociduo@vip.qq.com>
-yubozhao <yubz86@gmail.com>
-yuche <i@yuche.me>
 z <haig8@msn.com>
 zack <zxyah@126.com>
+Zack Craig <zack@zack6849.com>
+Zap <a124116186@qq.com>
 zerob4wl <zerob4wl@gmail.com>
+Zhang Zhi <fytriht@gmail.com>
 zhangpc <zhangpc@tenxcloud.com>
 zhaocai <lzc09008@gmail.com>
+Zheeeng <hi@zheeeng.me>
 zhujun24 <zhujun87654321@gmail.com>
 zhuyue <fuping.dfp@antfin.com>
 zilong <jzlxiaohei@163.com>
+Ziluo <gyfzzu@gmail.com>
 zinkey <yaya@uloveit.com.cn>
 zollero <corona7@163.com>
 zombiej <smith3816@gmail.com>
 zuiidea <zuiiidea@gmail.com>
-邦 <sqibang@gmail.com>
-爱but的苍蝇 <354788473@qq.com>
-拷钉 <41830859@qq.com>
-竹尔 <Juelchiang@gmail.com>
+रोहन मल्होत्रा <rohan.malhotra@adwyze.com>
+一喵呜 <hyb628@gmail.com>
+乔奕轩 <qiao_yixuan@163.com>
+低位 <zhujun87654321@gmail.com>
 偏右 <afc163@gmail.com>
-英布 <chaoren1641@gmail.com>
-朮厃 <cn.ah.liu@gmail.com>
-诸岳 <dengfuping_develop@163.com>
-逸达 <dqaria@gmail.com>
-蔡伦 <sliuqin@gmail.com>
-陆离 <surgesoft@gmail.com>
-愚道 <tingzhao.ytz@antfin.com>
-陈帅 <wasd2144@hotmail.com>
-松子 <window.pibarr@gmail.com>
 可乐 <zaxlct@foxmail.com>
+吕立青 <jimmy.jinglv@gmail.com>
 山客 <zeakhold@gmail.com>
-曾凯 <zengkai2009@foxmail.com>
-低位 <zhujun87654321@gmail.com>
+崔宏森 <948346354@qq.com>
 广彬-梁 <326741518@qq.com>
 徐坤龙 <272992168@qq.com>
-黄子毅 <576625322@qq.com>
+愚指导 <yutingzhao1991@sina.com>
+愚指导-TZ <yutingzhao1991@sina.com>
+愚道 <tingzhao.ytz@antfin.com>
+拷钉 <41830859@qq.com>
+曾凯 <zengkai2009@foxmail.com>
+朮厃 <cn.ah.liu@gmail.com>
+松子 <window.pibarr@gmail.com>
+柚子男 <yozman@sina.com>
+爱but的苍蝇 <354788473@qq.com>
+王集鹄 <wjhu111@21cn.com>
+白羊座小葛 <abeyuhang@gmail.com>
+竹尔 <Juelchiang@gmail.com>
 翁润雨 <593110501@qq.com>
-崔宏森 <948346354@qq.com>
+英布 <chaoren1641@gmail.com>
+蔡伦 <sliuqin@gmail.com>
+诸岳 <dengfuping_develop@163.com>
+超能刚哥 <margox@foxmail.com>
+逸达 <dqaria@gmail.com>
+邦 <sqibang@gmail.com>
 闲耘™ <hotoo.cn@gmail.com>
-一喵呜 <hyb628@gmail.com>
-吕立青 <jimmy.jinglv@gmail.com>
+陆离 <surgesoft@gmail.com>
+陈帅 <wasd2144@hotmail.com>
 隋鑫磊 <joshuasui@gmail.com>
-乔奕轩 <qiao_yixuan@163.com>
 马斯特 <sd4399340@126.com>
-王集鹄 <wjhu111@21cn.com>
-柚子男 <yozman@sina.com>
-愚指导 <yutingzhao1991@sina.com>
-愚指导-TZ <yutingzhao1991@sina.com>
-超能刚哥 <margox@foxmail.com>
 马金花儿 <o.o@mug.dog>
-रोहन मल्होत्रा <rohan.malhotra@adwyze.com>
-白羊座小葛 <abeyuhang@gmail.com>
+黄子毅 <576625322@qq.com>
```

Comment:
Could you revert this change? We will update it.

Reply:
Yes, of course. Should I remove myself, too?

Reply:
You can keep yourself.

Reply:
done, AUTHORS.txt of feature-3.8.0 and I kept myself in it.